dependency-check-suppressions.xml

```diff
@@ -31,4 +31,10 @@
         <vulnerabilityName>A prototype pollution vulnerability in handlebars is exploitable if an attacker can control the template</vulnerabilityName>
     </suppress>
 
+    <suppress>
+        <notes>Not used</notes>
+        <cve>CVE-2019-20444</cve>
+        <cve>CVE-2019-20445</cve>
+    </suppress>
```

Comment:
these should not be suppressed - I addressed the vulnerabilities in PR-229, if you merge `master` to your branch the warning should go away